src/adapters/commandLine.ts

```diff
@@ -1,10 +1,24 @@
 import { spawn } from 'child_process'
-import ICommandTrigger from '../ports/ICommandTrigger'
-import ProcessWrapper from '../types/processWrapper'
+import ICommandLine from '../ports/ICommandTrigger'
+import { AppComponents } from './../types'
+import { ConversionResult } from '../types/asset-converter'
 
-export default function createCommandTrigger(): ICommandTrigger {
-  function execute(command: string, args: string[]): ProcessWrapper {
-    return new ProcessWrapper(spawn(command, args))
+export default async function createCommandLineAdapter({ logs }: Pick<AppComponents, 'logs'>): Promise<ICommandLine> {
+  const logger = logs.getLogger('commandLineAdapter')
```

Comment:
Let's unify all logger names either on kebab-case names or camel-case (we're using kebab in another one further below). I personally prefer kebab, but up to you as long they're consistent.